docs/guides/tutorials/rest_apis_with_fastapi.rst

```diff
@@ -0,0 +1,535 @@
+.. _ref_guide_rest_apis_with_fastapi:
+
+===============================
+REST APIs with EdgeDB & FastAPI
+===============================
+
+EdgeDB can help you quickly build REST APIs in Python without getting into the rigmarole of using ORM libraries to handle your data effectively. Here, we'll be using `FastAPI <https://fastapi.tiangolo.com/>`_ to expose the API endpoints and EdgeDB to store the content.
```

Comment:
One nit: we limit line lengths to 79 characters max for all files in this repo.

Reply:
Done